### PR TITLE
adds first portfolio activity to structuring content

### PR DESCRIPTION
**Pull request by spitsfire**
it's currently missing a correct Ada C16 repo link

### CODE REVIEW

File: structuring-content/activity-personal-portfolio-1.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+### Setup
+
+Fork the following repo: [personal portfolio website](), then clone it to your local machine. From here you can begin creating the following pages based on the requirements below.
+
+### Requirements
+
+This portfolio should have the following pages:
+- `index.html`
+- `about.html`
+- `portfolio.html`
```
Comment (CheezItMan) [inline structuring-content/activity-personal-portfolio-1.md:20]: I suggest we use something like what we did in the JS sections.  Although it won't let me insert 3 backtics around it to make it look better. 

```suggestion
.
├── index.html
├── about.html
└── portfolio.html
```

File: structuring-content/activity-personal-portfolio-1.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+### Page Details
+- `index.html` must include your name somewhere.
+
+- `about.html` should include some information about you, your interests, background or similar. Only post what you're comfortable sharing.
+
+- `portfolio.html` must include information about the projects you've completed at Ada (or elsewhere) with links to GitHub repo if available, descriptions, images, etc.
```
Comment (CheezItMan) [inline structuring-content/activity-personal-portfolio-1.md:27]:
```suggestion
- `portfolio.html` must include information about several projects you've completed at Ada (or elsewhere) with links to the GitHub repo if available, descriptions, images, etc.
```

**Review by CheezItMan — CHANGES_REQUESTED** [review]
I suggest that we either comment out for this cohort or remove "Part 4: Recreate One of These Resumes" in the problemset.  Just because that's supposed to take "3 hours".  This covers that learning goal, and honestly does a better job.

Otherwise this is good, I would also suggest they employ an ordered or unordered list.

File: structuring-content/activity-personal-portfolio-1.md [hunk 3]
```diff
@@ -15,9 +15,10 @@ Fork the following repo: [personal portfolio website](), then clone it to your l
 ### Requirements
 
 This portfolio should have the following pages:
-- `index.html`
-- `about.html`
-- `portfolio.html`
+.
+├── index.html
+├── about.html
+└── portfolio.html
```
Comment (spitsfire) [inline structuring-content/activity-personal-portfolio-1.md:21]:
```suggestion
├── `index.html`
├── `about.html`
└── `portfolio.html`
```

**Review by CheezItMan — APPROVED** [review]

**Review by jericahuang — APPROVED** [review]

**Pull request merged**